lib/listudy_web/templates/study/show.html.eex

```diff
@@ -146,7 +145,10 @@
   i18n.fast = "<%= dgettext "study", "fast" %>";
   i18n.medium = "<%= dgettext "study", "medium" %>";
   i18n.instant = "<%= dgettext "study", "instant" %>";
+  i18n.comments_only_move = "<%= dgettext "study", "Comments: only played move" %>";
```

Comment:
Maybe "only my moves"?

Reply:
I had written "my moves" first, before I realized it's not a comment on **your** move, it's actually the **opponent's** move. It's the responses that are yours. "The played move" or "the last move" or something was the best I could come up with.